### ISSUE TITLE
Cannot scroll on query editor

### ISSUE THREAD

**Issue by AntoineAugusti** [getredash/redash#1024]
I cannot scroll on the query editor with a mouse. It seems that it works well with a touchpad... Re:dash 0.10.0+b1774

Demo URL: http://demo.redash.io/queries/1600/source
Video: http://sendvid.com/mo3jokk3

System details: OS X 10.11.3 / Chrome 50.0.2661.86 (reproduced normal and incognito mode)

**Comment by grzlybear** [comment]
I also have scrolling issues. I'm using a touchpad.
My issue is a bit different - the scroll seems to work both for the page and for the query editor in parallel - making long query editing quite difficult.

**Comment by arikfr** [comment]
Considering the massive changes to the query editor, I'm hoping this is no longer relevant.
